doc/rose-rug-files.html

```diff
@@ -522,17 +524,27 @@ <h3 class="alwayshidden">Other Useful Commands</h3>
       running:</p>
 
       <ul class="incremental">
-        <li><kbd>rose suite-scan</kbd>: list your running suites.</li>
-
-        <li><kbd>rose suite-gcontrol</kbd>: launch the <code>cylc gui</code>
-        for the suite in <var>$PWD</var>.</li>
-
-        <li><kbd>rose suite-shutdown</kbd>: shutdown the suite in
-        <var>$PWD</var> at the next opportunity.</li>
-
-        <li><kbd>rose suite-clean</kbd>: remove items created by <code>rose
-        suite-run</code> for the suite in <var>$PWD</var> and those on remote
-        job hosts.</li>
+        <li><kbd>rose suite-scan</kbd> or <kbd>cylc scan</kbd>: list your
+        running suites.</li>
+
+        <li><kbd>cylc gscan</kbd>: GTK version of <kbd>cylc scan</kbd>.</li>
+
+        <li><kbd>rose suite-gcontrol [--name=NAME]</kbd> or <kbd>cylc gui
```

Comment:
Not sure about telling people to run cylc gui like this as it needs the --host option to make it run on the correct server - if its not provided you (currently) get a "connection failed" dialog on opening of the cylc gui - or has one of your recent cylc changes altered this behaviour?